test_db.py

```diff
@@ -0,0 +1,231 @@
+import unittest
+import uuid
+from PIL import Image
+import io
+import base64
+import db.queries.user as user
+import db.queries.seed as seed
+import db.queries.picture as picture
+import metadata.picture_set as picture_set_data
+import metadata.picture as picture_data
+import db.__init__ as db
+
+"""
+This is a test script for the database packages. 
+It tests the functions in the user, seed and picture modules.
+"""
+
+
+def is_valid_uuid(val):
+    """
+    This validates if a given string is a UUID
+    """
+    try:
```

Comment:
In unittest you can use the `assertIsInstance()` function. If this is a validation function, I suggest to move it to metadata.validator module